tests/EntityManager.mongo.test.ts

```diff
@@ -846,6 +846,12 @@ describe('EntityManagerMongo', () => {
     book = (await orm.em.findOne(Book, book._id))!;
     expect(book.tags.count()).toBe(2);
 
+    // slice
+    expect(book.tags.slice()).toEqual(items);
+    expect(book.tags.slice(0,2)).toEqual(items);
+    expect(book.tags.slice(1)).toEqual(items[1]);
+    expect(book.tags.slice(0,1)).toEqual(items[0]);
```

Comment:
formatting, same for all the other tests

```suggestion
    expect(book.tags.slice(0, 2)).toEqual(items);
    expect(book.tags.slice(1)).toEqual(items[1]);
    expect(book.tags.slice(0, 1)).toEqual(items[0]);
```

and the assertions are most probably wrong, how can you compare an array of items to the item (the second line)? you need to compare arrays with arrays, even if they have a single item in them